app-tasks/Dockerfile

```diff
@@ -10,7 +10,6 @@ RUN set -ex \
                --shell /usr/sbin/nologin \
                airflow \
     && mkdir -p ${AIRFLOW_HOME} \
-    && chown airflow:airflow -R ${AIRFLOW_HOME} \
```

Comment:
I think that the `USER` directive below needs to be changed for this to work without task definition overrides.

Reply:
Changed that this morning and forgot to push :man_facepalming: